### PR TITLE
Map: doc update + fix regression on location drag icon

### PR DESCRIPTION
**Pull request by phweyland**
Updates map documentation with the recent changes.
Fixes also a regression of the last map update.

### CODE REVIEW

File: doc/usermanual/map/map.xml [hunk 1]
```diff
@@ -110,6 +110,27 @@
         onto the filmstrip.
       </para>
 
+      <para>
+        Images closed to each others are grouped under a single one. See map settings to
+        control the grouping as needed. The number displayed on bottom left of the images
```
Comment (TurboGit) [inline doc/usermanual/map/map.xml:115]: The number of images in the group is displayed on the bottom left of the thumb.

File: doc/usermanual/map/map.xml [hunk 2]
```diff
@@ -110,6 +110,27 @@
...
+        gives the number of images inside the group. White, it means all images of the
```
Comment (TurboGit) [inline doc/usermanual/map/map.xml:116]: A white border means...

File: doc/usermanual/map/map.xml [hunk 3]
```diff
@@ -110,6 +110,27 @@
...
+        group are exactly at the same location. Yellow, the images are approximately at the same place.
```
Comment (TurboGit) [inline doc/usermanual/map/map.xml:117]: A yellow border means the...

File: doc/usermanual/map/panels/locations.xml [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+        <listitem><para>
+          Edit location.
+        </para></listitem>
+
+        <listitem><para>
+          Delete location.
+        </para></listitem>
+
+        <listitem><para>
+          Update filmstrip. Filmstrip is fullfilled with the location's images.
```
Comment (TurboGit) [inline doc/usermanual/map/panels/locations.xml:112]: fulfilled (a single l) but I'm wondering if populated won't be better.

**Review by TurboGit — CHANGES_REQUESTED** [review]
Some comments. TIA

File: doc/usermanual/map/panels/locations.xml (hunk 4 above)
Comment (phweyland) [reply doc/usermanual/map/panels/locations.xml]: yes, it is.

File: doc/usermanual/map/map.xml (hunk 3 above)
Comment (phweyland) [reply doc/usermanual/map/map.xml]: done

File: doc/usermanual/map/map.xml [hunk 5]
```diff
@@ -110,6 +110,27 @@
         onto the filmstrip.
       </para>
 
+      <para>
+        Images closed to each others are grouped under a single one. See map settings to
+        control the grouping as needed. The number displayed on the bottom left of the thumb
+        gives the number of images inside the group. A white number means that all images of the
+        group are exactly at the same location. Otherwize the number is yellow.
```
Comment (TurboGit) [inline doc/usermanual/map/map.xml:117]: Otherwise (no z)

Comment (phweyland) [reply doc/usermanual/map/map.xml]: Done, sorry.

**Review by TurboGit — APPROVED** [review]
Thanks !

**Pull request merged**